site/content/extend/plugins/server/hello-world.md

```diff
@@ -1,12 +1,10 @@
 ---
-title: Quick Start (Hello, world!)
+title: Quick Start
 date: 2018-07-10T00:00:00-05:00
 subsection: Server Plugins
 weight: -10
 ---
 
-# Hello, world!
```

Comment:
Having "Quick Start" and "Hello world" in the title seams redundant to me. So I removed to "Hello, world".

But 1/5 what to do there.